_inc/client/plans/single-product-backup.jsx

```diff
@@ -36,6 +37,28 @@ export function SingleProductBackup( { plan, products, upgradeLinks, isFetchingD
 	);
 }
 
+function PromoNudge() {
+	const percent = 70;
+
+	return (
+		<div className="single-product-backup__promo">
+			<div className="single-product-backup__promo-star">
+				<img src={ `${ imagePath }/green-star.svg` } alt="" />
+				<span className="single-product-backup__promo-star-text">
+					{ __( 'Up to %(percent)d%% off!', { args: { percent } } ) }
```

Comment:
How about having this translation more flexible so that not only percentage values are accepted here?

We could do this by replacing the `percent` with a `discount` like:

```diff
- const percent = 70;
+ const discount = '70%';
```

and then the changing the translation to:

```diff
- { __( 'Up to %(percent)d%% off!', { args: { percent } } ) }
+ { __( 'Up to %(discount)s off!', { args: { discount } } ) }
```

Reply:
@delawski - Do we know if i18n will affect the positioning of the percentage signs, or is it universally on the right side? I wrote it like that so the translators can move/change the percentage.

Reply:
You are right, @ChaosExAnima! I just read about [some rules around percentage sign](https://en.wikipedia.org/wiki/Percent_sign). There are cases when it can be on the other side of the number (e.g. Turkish, Persian, Arabic).

Please disregard my comment and leave the code as is.